Dockerfile-alpine.template

```diff
@@ -77,6 +77,15 @@ RUN set -ex; \
 	mv /usr/src/GeoIPCity/GeoLite2-City.mmdb /usr/src/piwik/misc/GeoLite2-City.mmdb; \
 	rm -rf GeoIPCity*
 
+RUN set -ex; \
+	curl -fsSL -o EnvironmentVariables.tar.gz \
+		"https://github.com/matomo-org/plugin-EnvironmentVariables/archive/3.0.0.tar.gz"; \
```

Comment:
instead of hardcoding version numbers, would be good to use this link: https://plugins.matomo.org/api/2.0/plugins/EnvironmentVariables/download/latest

Reply:
fixed, thanks!